pkg/spell/spell.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2021 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package spell file defines a simple spelling checker for use in attribute errors
+// such as "no such field .foo; did you mean .food?".
```

Comment:
not sure if we took this file from starlark. if we did, lets mention that in a comment

Reply:
We did. I will.